src/libsystemd/sd-journal/journal-internal.h

```diff
@@ -79,7 +84,8 @@ struct sd_journal {
         OrderedHashmap *files;
         IteratedCache *files_cache;
         MMapCache *mmap;
-        Hashmap *newest_by_boot_id; /* key: boot_id, value: prioq, ordered by monotonic timestamp of last update */
+        NewestByBootId *newest_by_boot_id;
```

Comment:
please add a comment here what this is, i.e. a bisectable array, ordered by boot id